src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+    private static final String READ_ERROR_MSG = "Error: file not found: ";
+    private static final String WRITE_ERROR_MSG = "Cant write data to file: ";
```

Comment:
It is not mandatory to make constants for these strings. The main goal of constants is **to make code easier to read**.